napari/_qt/experimental/qt_monitor.py

```diff
@@ -0,0 +1,43 @@
+"""QtMonitor
+"""
+from qtpy.QtCore import QObject, QTimer
+
+from ...components.camera import Camera
+from ...components.experimental.monitor import monitor
+
+POLL_INTERVAL_MS = 33  # About 30Hz.
+
+
+class QtMonitor(QObject):
+    """Polls the monitor service.
+
+    This object is only created if NAPARI_MON is set.
+
+    To send data, it's only necessary to poll when the camera moves.
+    However to receive commands right now we need to poll all the time,
+    with a timer. We should be able to rid of timed polling eventually.
+
+    Parameters
+    ----------
+    parent : QObject
+        Parent Qt object.
+    """
+
+    def __init__(self, parent: QObject, camera: Camera):
+        super().__init__(parent)
+        camera.events.center.connect(self._on_camera_move)
```

Comment:
```suggestion
        camera.events.connect(self._on_camera_move)
```

This will connect to all events emitted by the camera - center, zoom, and angles which is ultimately I think what you want

Reply:
Great sounds good.